modules/simulation/DCSupplySimulator/main/power_supply_DCImpl.hpp

```diff
@@ -1,5 +1,5 @@
 // SPDX-License-Identifier: Apache-2.0
-// Copyright chargebyte GmbH and Contributors to EVerest
+// Copyright Pionix GmbH and Contributors to EVerest
```

Comment:
The module was written by Chargebyte, right?

Reply:
argh, we need to change the auto generator